### PR TITLE
Release 2.6.0

### PR DESCRIPTION
**Pull request by d4straub**
Summary of changes:
- Updated taxonomic databases (UNITE 9.0, PR2 5.0.0)
- Added more choice for taxonomic classification: phylogenetic placement, SINTAX
- Codon usage based filtering allows for additional QC of coding regions
- nf-core template update from 2.7.2 to 2.8

See CHANGELOG for detailed changes.

I will start the AWS full test dataset on branch dev. 
_Edit:_ started AWS full test dataset run on dev, results are slowly building up [here](https://nf-co.re/ampliseq/dev/results#ampliseq/results-55d72f40502b200fc39c09c358b26c58a93a9169/).
_Edit2:_ All successful!

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by sateeshperi — APPROVED** [review]
LGTM

**Comment by d4straub** [comment]
Thank you! Waiting for one more approval...

**Review by ramprasadn — APPROVED** [review]
LGTM! 🚀

**Comment by d4straub** [comment]
Thanks for the second review! I never had so fast 2 approvals I think.
I will wait for the AWS test to finish before making the release though.

File: conf/test_full.config
Comment (maxulysse) [inline conf/test_full.config:14]: This should go, otherwise we don't keep results

Comment (d4straub) [reply conf/test_full.config:14]: Thanks, but when do we not keep results? Because in nf-core/mag it seems to appear as well: https://github.com/nf-core/mag/blob/66cf53aff834d2a254b78b94fc54cd656b8b7b57/conf/test_full.config#L13 and it seems to keep results on the [website](https://nf-co.re/mag/results) (in case you mean that...).

**Pull request merged**
